### DISCUSSION TITLE
How can I create Side conversations (Child tickets)

### DISCUSSION THREAD

**Discussion by juanmanuelbasalo** [mozts2005/ZendeskApi_v2#521] [Features]
I need to create child tickets, investigating I realized the best was to use side_conversations that are displayed in the zendesk documentation, but I dont see it in the nugget package.

**Comment by mozts2005** [comment]
Can you share the zendesk api page for this part of the api?

**Reply by juanmanuelbasalo** [reply]
Good day Elizabeth, thanks a lot for answering.
Sure, the doc page I found on google is: **https://developer.zendesk.com/api-reference/ticketing/side_conversation/side_conversation/**
And the url looks like: **POST /api/v2/tickets/{ticket_id}/side_conversations**
I need it to create child tickets, I dont know if there is another way to create it with the nugget.

**Reply by mozts2005** [reply]
we would take as a community addition.

**Reply by juanmanuelbasalo** [reply]
Thanks Elizabeth. I need this to work a new project at my company, do you know how long will it take?. If not I can do it manually, just to know.

Thanks for your time.

**Reply by mozts2005** [reply]
someone from the community will need to do the work and submit it as a pull request. Given that I cannot say how long it will take. if you need fast, you could submit the changes needed as a pull request or your company could fund the changes by donating to the project.  for more about funded development feel free to email me.

**Reply by juanmanuelbasalo** [reply]
Awesome, gonna talk with my supervisor to check the course of action.
